### PR TITLE
Replace backslashes with forward slashes for filepaths on Windows

### PR DESCRIPTION
**Pull request by queengooborg**
This should fix the issue reported by @hamishwillee in https://github.com/mdn/browser-compat-data/pull/17278#issuecomment-1208799737.

### CODE REVIEW

**Comment by hamishwillee** [comment]
@queengooborg Exactly same behaviour  as before:  ran `npm install` then `npm test`

**Comment by queengooborg** [comment]
Argh...  I don't have a way to try out things in Windows, but it's definitely something that's related to the filepaths.

File: test/lint.ts [hunk 1]
```diff
@@ -54,8 +54,12 @@ const loadAndCheckFiles = async (...files: string[]): Promise<DataType> => {
         full: path.relative(process.cwd(), file),
         category: '',
       };
-      if (filePath.full.includes(path.sep)) {
-        filePath.category = filePath.full.split(path.sep)[0];
+      if (path.sep !== '/') {
+        // Normalize file paths for Windows users
+        filePath.full = filePath.full.replace(path.sep, '/');
```
Comment (hamishwillee) [inline test/lint.ts:59]: @queengooborg I added some crappy logging . It seems to be showing that this only replaces the first separator, and you have several.

```
Before replace: webextensions\manifest\storage.json 
After replace: webextensions/manifest\storage.json 
```

Could this be the cause? If not, add logging where you need and I'll run this for you.

Comment (queengooborg) [reply test/lint.ts:59]: Ack, right, I forgot that JavaScript's `.replace()` function is dumb... 🤦‍♀️

Comment (hamishwillee) [reply test/lint.ts]: Yeah, lets move to Python for all of this :-)

**Review by hamishwillee — APPROVED** [review]
Fix is good thanks. I can't merge though :-)

**Pull request merged**